File: ympa_fsharp_functions_server/src/model/OutletWorkingScheduleDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.OutletWorkingScheduleItemDTO

module OutletWorkingScheduleDTO =

  //#region OutletWorkingScheduleDTO

  [<CLIMutable>]
  type OutletWorkingScheduleDTO = {
    [<JsonProperty(PropertyName = "workInHoliday")>]
    WorkInHoliday : bool;
    [<JsonProperty(PropertyName = "scheduleItems")>]
    ScheduleItems : OutletWorkingScheduleItemDTO[];
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/OutletWorkingScheduleDTO.scala
package org.openapitools.server.model


/**
 * Список режимов работы точки продаж. 
 *
 * @param workInHoliday Признак, работает ли точка продаж в дни государственных праздников.  Возможные значения:  * `false` — точка продаж не работает в дни государственных праздников. * `true` — точка продаж работает в дни государственных праздников.  for example: ''null''
 * @param scheduleItems Список расписаний работы точки продаж.  for example: ''null''
*/
final case class OutletWorkingScheduleDTO (
  workInHoliday: Option[Boolean] = None,
  scheduleItems: Seq[OutletWorkingScheduleItemDTO]
)

